SeriesMaster-Frontend/App/Screens/DetailsScreen.js

```diff
@@ -36,10 +36,10 @@ const DetailsScreen = ({ route }) => {
   const [tvShow, setTvShow] = useState("");
 
   useEffect(() => {
-    fetchShows();
+    fetchShow();
   }, []);
 
-  async function fetchShows() {
+  async function fetchShow() {
```

Comment:
Not related to this change, but you could move `fetchShow` into `useEffect` to avoid creating a new function on every render because you only use it inside your effect.
In this case, you will get a warning that `route.params.id` is used in `useEffect` and should be added as a dependency. 
These makes sense, because you would trigger a fetch if `id` is changed.
Maybe you don't have this use case right now, but you could imagine this scenario:
You click on a show to see the details. `DetailsScreen` is rendered and fetch the show by `id`.
If you have a link to a similar show, you might redirect to a different `id`. React will notice that it's a `DetailsScreen` too and will reuse the component, but the `id` is changed. If you add `route.params.id` as useEffect dependency, the new show will be loaded.